test/utils/test_linear_cg.py

```diff
@@ -143,6 +143,26 @@ def test_batch_cg_with_tridiag(self):
                 approx_eigs = torch.linalg.eigvalsh(t_mats[j, i])
                 self.assertTrue(torch.allclose(eigs, approx_eigs, atol=1e-3, rtol=1e-4))
 
+    def test_batch_cg_init(self):
+        batch = 5
+        size = 100
+        matrix = torch.randn(batch, size, size, dtype=torch.float64)
+        matrix = matrix.matmul(matrix.mT)
+        matrix.div_(matrix.norm())
+        matrix.add_(torch.eye(matrix.size(-1), dtype=torch.float64).mul_(1e-1))
+
+        # Initial solve
+        rhs = torch.randn(batch, size, 50, dtype=torch.float64)
```

Comment:
The randomness here seems like it could easily result in a flaky test - have you run this in a stress test (e.g. a few hundred iterations of this) to check this doesn't happen? Alternatively you could fix the seed.

Reply:
Seems to me like the seed is already fixed: https://github.com/Turakar/linear_operator/blob/9b30670d55740499405c80ddeb9cc66fd36fd7df/test/utils/test_linear_cg.py#L18